compile-translations.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash -x
 
 ASKNOT_LOCALE_DIR=${ASKNOT_LOCALE_DIR:-l10n/fedora/locale}
+rm -f container/l10n.conf
 
 for locale in $(ls $ASKNOT_LOCALE_DIR/*.po); do
     echo $locale;
     locale=$(basename $locale)
     locale=${locale%.po};
     mkdir -p $ASKNOT_LOCALE_DIR/$locale/LC_MESSAGES/;
     msgfmt -o $ASKNOT_LOCALE_DIR/$locale/LC_MESSAGES/asknot-ng.mo $ASKNOT_LOCALE_DIR/$locale.po;
+
+    printf "RewriteCond %%{HTTP:Accept-Language} ^${locale//_/$'-'} [NC]\n" >> container/l10n.conf
+    printf "RewriteRule \"^/$\" \"/$locale/\" [L,R]\n\n" >> container/l10n.conf
 done
 
```

Comment:
This script is not run during the container image build so the l10n.conf file will not be available. I think we can just commit it to the git repo and update it when needed.

What do you think ?

Reply:
It is run during the container build? I think the .gitignore and this are correct.

Docker build runs build.sh:

https://github.com/fedora-infra/asknot-ng/blob/afb23e4db4bc7fdfaaecf6dac36b5963904ea81a/Dockerfile#L6

build.sh calls compile-translaions.sh:

https://github.com/fedora-infra/asknot-ng/blob/afb23e4db4bc7fdfaaecf6dac36b5963904ea81a/build.sh#L3

Reply:
> It is run during the container build? I think the .gitignore and this are correct.
> 
> Docker build runs build.sh:
> 
> https://github.com/fedora-infra/asknot-ng/blob/afb23e4db4bc7fdfaaecf6dac36b5963904ea81a/Dockerfile#L6
> 
> build.sh calls compile-translaions.sh:
> 
> https://github.com/fedora-infra/asknot-ng/blob/afb23e4db4bc7fdfaaecf6dac36b5963904ea81a/build.sh#L3

Ah nice, then let's merge this :-) thanks